### PR TITLE
Add method takeScreenshot ot WindowManager helper.

### PR DESCRIPTION
**Pull request by TarasMalynovskyy**

### CODE REVIEW

File: src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java [hunk 1]
```diff
@@ -34,27 +42,55 @@ public GoogleLoginPage(WebDriver driver) {
     }
 
     public WebElement getEmailField() {
+        if (GreenCityTestRunner.CHROME_HEADLESS_OPTION) {
+            wait = new WebDriverWait(driver, 5);
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java:46]: change to ternary operator

File: src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java [hunk 2]
```diff
@@ -34,27 +42,55 @@ public GoogleLoginPage(WebDriver driver) {
...
 
         return emailNextButton = driver.findElement(By.xpath(EMAIL_NEXT_BUTTON_XPATH));
     }
 
     public WebElement getPasswordField() {
+        if (GreenCityTestRunner.CHROME_HEADLESS_OPTION) {
+            wait = new WebDriverWait(driver, 5);
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id(PASSWORD_FIELD_ID_HEADLESS)));
+
+            return emailField = driver.findElement(By.id(PASSWORD_FIELD_ID_HEADLESS));
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java:77]: fix

File: src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java [hunk 3]
```diff
@@ -34,27 +42,55 @@ public GoogleLoginPage(WebDriver driver) {
...
 
         return emailField = driver.findElement(By.id(EMAIL_FIELD_ID));
     }
 
     public WebElement getEmailNextButton() {
+        if (GreenCityTestRunner.CHROME_HEADLESS_OPTION) {
+            wait = new WebDriverWait(driver, 5);
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id(EMAIL_NEXT_BUTTON_ID_HEADLESS)));
+
+            return emailField = driver.findElement(By.id(EMAIL_NEXT_BUTTON_ID_HEADLESS));
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java:63]: fix

File: src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java [hunk 4]
```diff
@@ -34,27 +42,55 @@ public GoogleLoginPage(WebDriver driver) {
...
 
         return passwordField = driver.findElement(By.xpath(PASSWORD_FIELD_XPATH));
     }
 
     public WebElement getPasswordNextButton() {
+        if (GreenCityTestRunner.CHROME_HEADLESS_OPTION) {
+            wait = new WebDriverWait(driver, 5);
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.id(PASSWORD_NEXT_BUTTON_ID_HEADLESS)));
+
+            return emailField = driver.findElement(By.id(PASSWORD_NEXT_BUTTON_ID_HEADLESS));
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java:91]: fix

File: src/test/java/com/softserve/edu/greencity/ui/tests/GreenCityTestRunner.java [hunk 5]
```diff
@@ -33,6 +33,7 @@ public void setUpBeforeClass() {
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.setHeadless(CHROME_HEADLESS_OPTION);
         chromeOptions.addArguments("--lang=" + CHROME_LANGUAGE_OPTION);
+        chromeOptions.addArguments("--start-maximized");
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/tests/GreenCityTestRunner.java:36]: delete

File: src/test/java/com/softserve/edu/greencity/ui/tools/WindowManager.java [hunk 6]
```diff
@@ -19,4 +25,15 @@ public void changeWindowWidth(int width) {
     public void maximizeWindow() {
         driver.manage().window().maximize();
     }
+
+    public static void takeScreenshot(WebDriver driver, String pathToStore) {
```
Comment (lhalam) [inline src/test/java/com/softserve/edu/greencity/ui/tools/WindowManager.java:29]: add java doc

File: src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java (hunk 1 above)
Comment (TarasMalynovskyy) [reply src/test/java/com/softserve/edu/greencity/ui/pages/cabinet/GoogleLoginPage.java:46]: Fixed

**Pull request merged**